### PR TITLE
Release/minor

### PR DESCRIPTION
**Pull request by nick-armstrong-helix**
## Background
The build system is terribly fragile... Doing this to promote the main orb to prod.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Initial Release
 
 
-[1.0.0]: GITHUB TAG URL
+[1.0.0]: GITHUB TAG URL
```
Comment (JackyLiHelix) [inline CHANGELOG.md:19]: extra spaces?

Also missing linebreak at EOF

**Review by JackyLiHelix — APPROVED** [review]
Why is there extra space

File: CHANGELOG.md (hunk 1 above)
Comment (nick-armstrong-helix) [reply CHANGELOG.md:19]: Yeah. The build system was ignoring my empty commits. It required a file diff...

Comment (JackyLiHelix) [reply CHANGELOG.md:19]: time to start adding ASCII art

Comment (nick-armstrong-helix) [reply CHANGELOG.md:19]: foreal

Comment (JackyLiHelix) [reply CHANGELOG.md:19]:
```
 _  _  __   __     __  
( \/ )/  \ (  )   /  \ 
 )  /(  O )/ (_/\(  O )
(__/  \__/ \____/ \__/ 
```

**Pull request merged**